ouroboros-consensus/src/Ouroboros/Consensus/Protocol/Abstract.hs

```diff
@@ -226,16 +229,15 @@ class ( Show (ChainState    p)
 
   default preferCandidate :: HasHeader hdr
                           => NodeConfig p
-                          -> AnchoredFragment hdr      -- ^ Our chain
-                          -> AnchoredFragment hdr      --` ^ Candidate
+                          -> hdr      -- ^ Our chain
+                          -> hdr      --` ^ Candidate
                           -> Bool
-  preferCandidate _ ours cand =
-    AF.compareHeadBlockNo cand ours == GT
+  preferCandidate _ ours cand = blockNo cand > blockNo ours
```

Comment:
Ah, this is nice :relieved:

Reply:
That will conflict with PR 1353, which we're hoping to merge soon and currently has

```
-  preferCandidate _ ours cand =
-    AF.compareHeadBlockNo cand ours == GT
+  preferCandidate cfg ours cand =
+    compareCandidates cfg ours cand == LT
```

(Note the unflip of the arguments and the switch to `compareCandidates`.)

Reply:
I see. Not a fundamental problem though is it? The equivalent of that code can still be done with this new approach also?

Reply:
They're compatible. I was just calling out an anticipated `git` conflict, but forgot to add "`git`".

Reply:
Ok, good :)